flytekit/remote/remote_fs.py

```diff
@@ -278,6 +231,7 @@ async def _put(
         res = await super()._put(lpath, REMOTE_PLACEHOLDER, recursive, callback, batch_size, **kwargs)
         if isinstance(res, list):
             res = self.extract_common(res)
+        FlytePathResolver.add_mapping(rpath.strip("/"), res)
```

Comment:
when is this necessary?   will this break on windows?

Reply:
Let's use `os.path.sep` instead?

Reply:
done